lib/internal/test_runner/runner.js

```diff
@@ -203,10 +199,15 @@ class FileTest extends Test {
     this.reportStarted();
     this.#handleReportItem(ast);
   }
+  reportStarted() {}
   report() {
-    this.reportStarted();
+    if (!this.skipReporting) {
```

Comment:
Since `skipReporting` is a getter that is used a couple times in this function, I think we should assign it to a variable here. Possibly even consider making `skipReporting` a function so that it's clear why we'd want to store the result in a variable.